.github/workflows/builder.yaml

```diff
@@ -79,15 +79,20 @@ jobs:
       - name: Check if add-on should be built
         id: check
         run: |
-          if [[ "${{ steps.info.outputs.architectures }}" =~ ${{ matrix.arch }} ]]; then
-             echo "build_arch=true" >> $GITHUB_OUTPUT;
-             echo "image=$(echo ${{ steps.info.outputs.image }} | cut -d'/' -f3)" >> $GITHUB_OUTPUT;
-             if [[ -z "${{ github.head_ref }}" ]] && [[ "${{ github.event_name }}" == "push" ]]; then
-                 echo "BUILD_ARGS=" >> $GITHUB_ENV;
-             fi
-           else
-             echo "${{ matrix.arch }} is not a valid arch for ${{ matrix.addon }}, skipping build";
-             echo "build_arch=false" >> $GITHUB_OUTPUT;
+          if [[ "${{ steps.info.outputs.image }}" != "null" ]]; then
```

Comment:
Per @ludeeus 's suggestion, moved the check logic that was initially in the `Build ${{ matrix.addon }} add-on` step to here. The info step returns the string `null` if image is not defined. If that's returned we continue on with the checks, else we set `build_arch=false` and move on.